XenModel/Messages.resx

```diff
@@ -6235,6 +6235,11 @@ Note: Any custom color selections will remain unchanged</value>
   <data name="GUEFI_SECURE_BOOT_MODE_LABEL" xml:space="preserve">
     <value>UEFI &amp;Secure Boot</value>
   </data>
+  <data name="GUI_NOT_COMPATIBLE" xml:space="preserve">
+    <value>This version of [XenCenter] can only connect to [XenServer] 7.1.2 and 8.2+ versions and cannot connect to {0}.

```

Comment:
The correct  product names and versions would be: XenServer 7.1 CU2 and Citrix Hypervison 8.2 and greater

Reply:
Fixed.